src/components/visualizations/chart/highcharts/plugins/dataLabelsColors.ts

```diff
@@ -51,6 +51,40 @@ function setBarDataLabelsColor(chart: any) {
     });
 }
 
+function setColumnDataLabelsColor(chart: any) {
+    const points = getVisiblePointsWithLabel(chart);
+
+    return points.forEach((point: any) => {
+        const labelDimensions = getDataLabelAttributes(point);
+        const columnDimensions = getShapeAttributes(point);
+        const columnTop = columnDimensions.y + columnDimensions.height;
+        const columnDown = columnDimensions.y;
+        const labelDown = labelDimensions.y;
+
+        if (point.negative) {
+            if (labelDown < columnDown) {
+                setWhiteColor(point);
+            } else {
+                setBlackColor(point);
+            }
+        } else {
+            if (!isStacked(chart)) {
+                if (labelDown < columnTop) {
```

Comment:
Consider "labelDown > columnTop" and then you can call inside that block setWhiteColor(point). Then you can create a function with one parameter (it will be a condition) and the body of the function will be the same.

Reply:
Done. Created global function to change colors.